Dockerfile

```
@@ -17,7 +17,6 @@ COPY internal/ internal/
 COPY pkg/ pkg/
 COPY versions.txt versions.txt
 
-ARG TARGETARCH=amd64
```

```suggestion
ARG TARGETARCH
```
would it make sense to remove `=amd64`?

---

Then we would have to pass the arch as argument which should not be necessary

---

But without we are not able to build it for a different target architecture, right?

---

we are via docker buildx